content/2.components/photo-gallery.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Photo Gallery
+description: Showcase your team with pride using our stunning Photo Gallery Component.
+navBadges:
+  - value: New
+    type: lime
+---
+
+::ComponentLoader{label="Preview" componentName="PhotoGalleryDemo" type="examples" id="photo-gallery"}
+::
+
+## API
+
+| Prop Name        | Type     | Default             | Description                                            |
+| ---------------- | -------- | ------------------- | ------------------------------------------------------ |
+| `items`          | `[]`     | `"[{src: string}]"` | Pass items / image src to animate                      |
```

Comment:
Seems like the type and default value information is swapped here.

Reply:
Done @rahul-vashishtha